components/Banner.tsx

```diff
@@ -66,14 +68,21 @@ const Banner = ({ bannerHeight, setBannerHeight }: {
 
 	useEffect(() => {
 		const determineSituation = (): BannerSituation => {
+            const isMobile = /Android|webOS|iPhone|iPad|iPod|BlackBerry|IEMobile|Opera Mini/i.test(navigator.userAgent);
```

Comment:
The userAgent property of the navigator object has a string that gives the user info about the browser, operating system, and device sent by the browser to the server. The test function is called on that expression, so if the device is any one of the following, it returns true, otherwise false.

Reply:
Is it possible that it fails? What is the error case?

Reply:
I think it might fail for smart watches/TVs that use iOS or some other OS.